src/events/crawler/scrapers/USA/SC/index.js

```diff
@@ -1,89 +1,69 @@
+const { Builder, By, until } = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
 
 const scraper = {
   state: 'SC',
   country: 'USA',
-  url:
-    'https://services2.arcgis.com/XZg2efAbaieYAXmu/arcgis/rest/services/COVID19_County_View/FeatureServer/0/query?f=json&where=Confirmed%20%3E%200&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*&orderByFields=Confirmed%20desc&resultOffset=0&resultRecordCount=1000&cacheHint=true',
-  type: 'json',
+  url: 'https://www.scdhec.gov/monitoring-testing-covid-19',
+  type: 'table',
   aggregate: 'county',
 
-  _counties: [
-    'Abbeville County',
-    'Aiken County',
-    'Allendale County',
-    'Anderson County',
-    'Bamberg County',
-    'Barnwell County',
-    'Beaufort County',
-    'Berkeley County',
-    'Calhoun County',
-    'Charleston County',
-    'Cherokee County',
-    'Chester County',
-    'Chesterfield County',
-    'Clarendon County',
-    'Colleton County',
-    'Darlington County',
-    'Dillon County',
-    'Dorchester County',
-    'Edgefield County',
-    'Fairfield County',
-    'Florence County',
-    'Georgetown County',
-    'Greenville County',
-    'Greenwood County',
-    'Hampton County',
-    'Horry County',
-    'Jasper County',
-    'Kershaw County',
-    'Lancaster County',
-    'Laurens County',
-    'Lee County',
-    'Lexington County',
-    'McCormick County',
-    'Marion County',
-    'Marlboro County',
-    'Newberry County',
-    'Oconee County',
-    'Orangeburg County',
-    'Pickens County',
-    'Richland County',
-    'Saluda County',
-    'Spartanburg County',
-    'Sumter County',
-    'Union County',
-    'Williamsburg County',
-    'York County'
-  ],
-
   async scraper() {
-    const data = await fetch.json(this.url);
-    let counties = [];
+    const counties = [];
+    const driver = await new Builder()
+      .setChromeOptions(new chrome.Options().headless().windowSize({ width: 2000, height: 2000 }))
+      .forBrowser('chrome')
+      .build();
 
-    for (const record of data.features) {
-      const rec = record.attributes;
+    try {
+      await driver.get(this.url);
 
-      const county = geography.addCounty(rec.NAME);
-      const cases = rec.Confirmed;
-      const deaths = rec.Death;
-      const recovered = rec.Recovered;
+      // Give time for iframes to load (anything more deterministic doesn't seem to work)
+      await driver.sleep(5000);
 
-      counties.push({
-        county,
-        cases,
-        deaths,
-        recovered
-      });
-    }
+      // kill the annoying header
+      await driver.executeScript("document.querySelector('header').remove()");
+
+      // Switch to iframe
+      let ifr = driver.wait(until.elementLocated(By.css('iframe')), 10000);
+      await driver.switchTo().frame(ifr);
+
+      // Switch to iframe
+      ifr = driver.wait(until.elementLocated(By.css('iframe')), 5000);
+      await driver.switchTo().frame(ifr);
```

Comment:
Can we just get the URL of the iframe and go directly to it?

Reply:
I have considered that.  I'm worried that the iframes's url might be dynamically created.
Do you think that url will be stable?

I have tried some like:
- find the iframe and get the URL
- do a new driver.get() with the iframe's url.

Reply:
I finally wised up and found the restful request that fetched the data for SC, only to find that I was beat to the punch.